content/en/localstack/limitations.md

```diff
@@ -30,3 +30,40 @@ Limitations that may occur because of third party integrations behavior.
 By default, stacks with validated certificates may not be deployed using the `local` lambda executor.
 This originates from the way how CDK ensures the certificate is ready - it creates a single-file lambda function with a single dependency on `aws-sdk` which is usually preinstalled and available globally in lambda runtime.
 When this lambda is executed locally from the `/tmp` folder, the package can not be discovered by Node due to the way how Node package resolution works.
+
+### DNS Rebind Protection
+
+For certain LocalStack features it is necessary that the DNS resolves to the local network.
+For example, OpenSearch clusters which are created by LocalStack can be reached via `<domain-name>.<region>.opensearch.localhost.localstack.cloud`.
```

Comment:
A, even more important, issue of this is S3 with the usage of virtual-host based addressing (since path-based is deprecated). Maybe we could add a line about this, since this might come up more often?

Reply:
Thanks for the hint, it mentioned S3 and ElasticSearch as well.